pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/MedicalRecordUtil.java

```diff
@@ -173,19 +176,16 @@ public static void writeInstitutionIntoDb(@NonNull Connection conn, @NonNull Str
         }
     }
 
-    private static void insertInstitution(@NonNull Connection conn, @NonNull PreparedStatement insertInstitution, @NonNull String id,
-                                          boolean setDuplicateFlag) {
+    private static void insertInstitution(@NonNull Connection conn, @NonNull PreparedStatement insertInstitution, String participantId,
+                                          String ddpParticipantId, String instanceName) {
```

Comment:
what is the difference between `participantId` and `ddpParticipantId`? does `participantId` come from some internal table which keeps data of participants?